tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -0,0 +1,150 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import re
+import sys
+from pathlib import Path
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="Create ORT Extensions nuget spec script", usage=""
+    )
+    # Main arguments
+    parser.add_argument("--package_name", required=True, help="ORT package name. Eg: ort-extensions-nuget")
+    parser.add_argument("--package_version", required=True, help="ORT package version. Eg: 0.7.0")
+    parser.add_argument("--native_build_path", required=True, help="Native build output directory.")
+    parser.add_argument("--sources_path", required=True, help="Extensions source code root.")
+
+    return parser.parse_args()
+
+def generate_id(line_list, package_name):
+    line_list.append("<id>" + package_name + "</id>")
+
+
+def generate_version(line_list, package_version):
+    line_list.append("<version>" + package_version + "</version>")
+
+
+def generate_authors(line_list, authors):
+    line_list.append("<authors>" + authors + "</authors>")
+
+
+def generate_owners(line_list, owners):
+    line_list.append("<owners>" + owners + "</owners>")
+
+
+def generate_description(line_list, package_name):
+    line_list.append("<description>ONNX Runtime Extensions NuGet Package</description>")
+
+
+def generate_copyright(line_list, copyright):
+    line_list.append("<copyright>" + copyright + "</copyright>")
+
+
+def generate_tags(line_list, tags):
+    line_list.append("<tags>" + tags + "</tags>")
+
+
+def generate_license(line_list):
+    line_list.append('<license type="file">LICENSE.txt</license>')
+
+
+def generate_project_url(line_list, project_url):
+    line_list.append("<projectUrl>" + project_url + "</projectUrl>")
+
+
+def generate_repo_url(line_list, repo_url, commit_id):
+    line_list.append('<repository type="git" url="' + repo_url + '"' + ' commit="' + commit_id + '" />')
+
+def generate_dependencies(line_list):
+    dependencies_list = ['<group targetFramework="native">']
+
+    # Update dependencies as needed here:
+    dependencies_list.append('<dependency id="blingfire" version="0831265c1aca95ca02eca5bf1155e4251e545328"/>')
```

Comment:
ort-extensions dll statically linked any dependency so it doesn't depend any 3rd party DLL here.

Reply:
perfect, thanks! will remove them in the next commit.

Reply:
Before completing this script, can we check in nuspec file of amd64 native binary only?

Reply:
sure - was initially motivated to write the script because generating the files list is easier with a script because we can do it recursively, would take a lot longer to just manually add the files list.